docs/product-getting-started-guide/src/main/asciidoc/task_business-rules.adoc

```diff
@@ -9,102 +9,108 @@ The following chapter instructs you how to create business rules in {PRODUCT} us
 +
-* *Column header (description)*: `Annual Income (Less Than Or Equal To)`
+. Move *annualincome..* to the *Conditions* section again.
+. From *Conditions*, select *annualincome*, then enter:
++
+* *Column header (description)*: `Annual Income (Less Than or Equal To)`
 * *Operator*: `less than or equal to`
-.. From *Available patterns*, select *Property*. From *Available fields*, select *saleprice*, and place it into *Conditions*.
-.. From *Conditions*, select *saleprice*, then enter the following: 
++
+. From *Available patterns*, select *Property*,  then from *Available fields*, select *saleprice:Whole number (interger)* and then move it to the *Conditions* section.
+. From *Conditions*, select *saleprice*, then enter:
 +
 * *Column header (description)*: `Sale Price (Less Than)`
 * *Operator*: `less than`
-.. From *Available fields*, select *age*, and place it into *Conditions*. 
-.. From *Conditions*, select *age*, then enter the following: 
+. From *Available fields*, select *age:Whole number (interger)*, and move it in to the *Conditions* section.
+. From *Conditions*, select *age*, then enter:
 +
 * *Column header (description)*: `Property Age (Less Than)`
 * *Operator*: `less than`
-.. From *Available fields*, select *locale*, and place it into *Conditions*. 
-.. From *Conditions*, select *locale*, then enter the following: 
+. From *Available fields*, select *locale:Text*, and move it to the *Conditions* section.
+. From *Conditions*, select *locale*, then enter:
 +
 * *Column header (description)*: `Location`
 * *Operator*: `equal to`
 * *(optional) value list*: `Urban,Rural`
-. Click *Next*, then click *Next*.
-. In the *Define actions to insert new Facts\Patterns* part:
-.. From *Available patterns*, select *Application* and place it into *Chosen patterns*.
-.. From *Chosen patterns*, select *Application*. Then, from *Available fields*, select *mortgageamount* and place it into *Chosen fields*.
-.. From *Chosen fields*, select *mortgageamount*, then enter the following:
+. Click *Next*.
+
+. Click *Add Actions to insert Facts*, select *Application* and move it to the *Chosen patterns* section.
+. From *Available fields*, select *mortgageamount:Whole number (integer)*, and move it to the *Chosen fields* section.
+.. From *Chosen fields*, select *mortgageamount*, then enter:
 +
 * *Column header (description)*: `Mortgage Amount`
 . Click *Finish*.
-. Click *Decision table* -> *New column*, select *include advanced options*, select *Add a new Attribute column*, then click *Next*.
+. Click *Save*, then *Save*, to confirm your changes.
+. Click *Insert > Append row* and click *Save*.
+. Click *Add Column*.
+. From the *Add a new column* window, select *New Column*.
+. Select *Include advanced options*, then select *Add a new Attribute column*.
 . Select *Ruleflow-group*, then click *Finish*.
-. Click *Insert* -> *Append row*. Repeat this step once.
-. Fill out the table as follows:
+. Fill out the table as shown below:
```

Comment:
IBM - "below" do not use to indicate a relative location in a document. You can use "following" to refer to a table though, so perhaps "Fill out the table as shown in the following example:" or whatever.